test/embind/embind_test.cpp

```diff
@@ -2843,7 +2843,7 @@ EMSCRIPTEN_BINDINGS(noncopyable) {
         .function("method", &Noncopyable::method)
         ;
 
-    function("getNoncopyable", &getNoncopyable);
+    function("getNoncopyable", &getNoncopyable, return_value_policy::take_ownership());
```

Comment:
To make sure I understand, this is a necessary change because this is non-copyable, and the default return value policy would copy?

Reply:
Missed this comment. Yes that's correct. This will be a breaking change, though at least the compiler will catch it and I think it's pretty uncommon.